dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerTest.java

```diff
@@ -613,6 +613,26 @@ void getTrackedEntityReturnsCsvGZipFormat() {
         () -> assertNotNull(response.content(ContextUtils.CONTENT_TYPE_CSV_GZIP)));
   }
 
+  @Test
+  void shouldGetSoftDeletedEnrollmentsWhenIncludeDeletedIsTrue() {
+    JsonList<JsonTrackedEntity> json =
+        GET(
+                "/tracker/trackedEntities?trackedEntities={id}&fields=enrollments&includeDeleted=true",
+                "woitxQbWYNq")
+            .content(HttpStatus.OK)
+            .getList("trackedEntities", JsonTrackedEntity.class);
+
+    JsonList<JsonEnrollment> enrollments = json.get(0).getEnrollments();
+    enrollments.forEach(
+        en -> {
+          assertEquals(EnrollmentStatus.CANCELLED.name(), en.getStatus());
```

Comment:
do we set the status to cancelled on delete?

Reply:
Yes :man_shrugging: